vnext/ReactUWP/ReactUWP.vcxproj.filters

```diff
@@ -627,6 +630,10 @@
     <ClInclude Include="Utils\ResourceBrushUtils.h">
       <Filter>Utils</Filter>
     </ClInclude>
+    <ClInclude Include="Utils\StandardControlResourceKeyNames.h" />
```

Comment:
Do you forgot to check in StandardControlResourceKeyNames.h or it's in other project?